### DISCUSSION TITLE
视频回放出现问题

### DISCUSSION THREAD

**Discussion by StarLiver** [netless-io/flat#1794] [Q&A] [answered]
升级到最新版本之后，之前版本的课堂回放好像有问题，中间的书写区域一直不变，只有一个我打开的课件的画面，这个有办法解决嘛？

**Comment by Leooeloel** [comment, marked as answer]
感谢反馈，Flat 新版本为了提升代码的可维护性、使用流畅度升级了 WindowManager 这个核心库，所带来的问题就是这次更新是断代更新。

以后我们会尽量减少这种改变，另外我们有一个产品动态交流群，会及时向用户同步更新信息，感兴趣可以添加 https://www.yuque.com/leooel/ec1kmm/vmsolg
